shared/Reusable queries for data extraction/query-admissions-covid-utilisation.sql

```diff
@@ -3,8 +3,9 @@
 --└────────────────────────────────────┘
 
 -- OBJECTIVE: To classify every admission to secondary care based on whether it is a COVID or non-COVID related.
---						A COVID-related admission is classed as an admission within 4 weeks after, or up to 2 weeks before
---						a positive test.
+
+-- NOTE: A COVID-related admission is classed as an admission within 4 weeks after, or up to 2 weeks before
```

Comment:
This is a great idea... but this affects the `generateReusableQueryDocs` function within the file `scripts/docs.js` which creates the file `docs/Reusable-SQL-query-reference.md`. That script parses the headers of these files to pull out the information. It expects an 'OBJECTIVE', but it doesn't expect a 'NOTE' so when you regenerate the document (using `npm start`) this notes disappears. When you've got nodejs installed you can have a go at correcting this.